gent_base.theme

```diff
@@ -460,10 +489,10 @@ function gent_base_preprocess_field__field_documents(&$variables) {
   if (isset($variables['documents_invisible'])) {
     $extra_documents_count = count($variables['documents_invisible']);
     if ($extra_documents_count === 1) {
```

Comment:
Use `\Drupal::translation()->formatPlural()` as plurals can be different based on the language...

See https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21StringTranslation%21TranslationManager.php/function/TranslationManager%3A%3AformatPlural/8.7.x